src/pages/global.tsx

```diff
@@ -7,11 +8,44 @@ import { componentSpacing, maxTextWidth } from 'theme/metrics';
 import trpc, { trpcNext } from 'trpc';
 import moment from 'moment-timezone';
 
+type MeetingSlot = {
```

Comment:
please define types shared between frontend and backend in src/shared using zod. See examples in src/shared/Kudos.ts and other files in the same folder